cypress/integration/Dialog.feature

```diff
@@ -0,0 +1,116 @@
+Feature: Dialog component
+  I want to change Alert component properties
+
+  Background: Open Dialog component page
+    Given I open "Dialog" component page
+
+  @positive
+  Scenario Outline: Set height for Dialog dialog
```

Comment:
I'm not sure how useful it is to test that a prop like this is set. 

It might be better to test that the element renders at the correct height?

Reply:
When test will set height then it verifies if height is styled
here example
dialogPreview().should('have.attr', 'style').should('contain', `min-height: ${height}px`)
is this ok?